pg_spot_operator/operator.py

```diff
@@ -59,13 +60,15 @@
 VM_KEEPALIVE_SCANNER_INTERVAL_S = 60
 ACTION_HANDLER_TEMP_SPACE_ROOT = "~/.pg-spot-operator/tmp"
 ANSIBLE_DEFAULT_ROOT_PATH = "./ansible"
+DEFAULT_CONFIG_DIR = "~/.pg-spot-operator"
```

Comment:
shouldn't all these constants go into the `constants.py` for import into other files?

Reply:
Saw that you actually moved that in your PR so will just merge that